src/main/java/ch/wisv/areafiftylan/security/authentication/AuthenticationServiceImpl.java

```diff
@@ -68,4 +70,10 @@ public void removeAuthToken(String xAuth) {
         token.revoke();
         authenticationTokenRepository.saveAndFlush(token);
     }
+
+    @Override
+    public void removeAllAuthTokens() {
+        authenticationTokenRepository.deleteAll();
```

Comment:
Shouldn't we `revoke()` all non-revoked tokens instead of dropping database contents? Seems like this is clearing log-in history, which we may not want.

Reply:
I don't think we should rely on persisted authenticationTokens for login history. If we want to keep tabs on that, this would be the place to log it: https://github.com/AreaFiftyLAN/lancie-api/blob/master/src/main/java/ch/wisv/areafiftylan/security/JsonLoginAuthenticationAttemptHandler.java#L49